packages/moonstone/PictureInGraphics/PictureInGraphics.js

```diff
@@ -0,0 +1,178 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import Pure from '@enact/ui/internal/Pure';
+import {compareSources} from '@enact/moonstone/VideoPlayer/util';
+import Media from '@enact/ui/Media';
+import Slottable from '@enact/ui/Slottable';
+import Spottable from '@enact/spotlight/Spottable';
+
+import Image from '../Image';
+import Skinnable from '../Skinnable';
+import {MarqueeController, Marquee} from '../Marquee';
+
+import css from './PictureInGraphics.less';
+
+const defaultPlaceholder =
+	'data:image/svg+xml;charset=utf-8;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC' +
+	'9zdmciPjxyZWN0IHdpZHRoPSIxMDAlIiBoZWlnaHQ9IjEwMCUiIHN0cm9rZT0iIzU1NSIgZmlsbD0iI2FhYSIg' +
+	'ZmlsbC1vcGFjaXR5PSIwLjIiIHN0cm9rZS1vcGFjaXR5PSIwLjgiIHN0cm9rZS13aWR0aD0iNiIgLz48L3N2Zz' +
+	'4NCg==';
+
+class PictureInGraphicsBase extends React.Component {
+	static propTypes = {
+		/**
+		 * Any children `<source>` tag elements will
+		 * be sent directly to the `<video>` element as video sources.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		source: PropTypes.node.isRequired,
+
+		/**
+		 * Image path for image overlay
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		imageOverlaySrc: PropTypes.string,
+
+		/**
+		 * By default, the video will start playing immediately after it's loaded, unless this is set.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		noAutoPlay: PropTypes.bool,
```

Comment:
Is there a use case where PIG wants to not autoplay upon its load? We do not offer to control the video state in any case, so I'm not sure how this can be ever useful when it is set to `true`.